crates/bin/pcli/src/command/tx/liquidity_position.rs

```diff
@@ -8,7 +8,7 @@ use penumbra_dex::{
     },
     TradingPair,
 };
-use rand_core::CryptoRngCore;
+use rand_core::OsRng;
```

Comment:
I wonder if we might not be able to make this use a generic RNG for easier testing